src/dlgprefcontrolsdlg.ui

```diff
@@ -203,7 +203,7 @@
         <item row="2" column="0">
          <widget class="QLabel" name="labelKeylockMode">
           <property name="text">
-           <string>Key lock mode</string>
+           <string>Keylock mode</string>
```

Comment:
as well as "replay gain", "United States", ... despite they are all written in one word ;)